packages/editor-ui/src/stores/workflows.store.ts

```diff
@@ -551,6 +551,9 @@ export const useWorkflowsStore = defineStore(STORES.WORKFLOWS, {
 			if (this.workflowsById[workflowId]) {
 				this.workflowsById[workflowId].active = true;
 			}
+			if (workflowId === this.workflow.id) {
+				this.setActive(true);
+			}
```

Comment:
And the changes in here perhaps?

Reply:
See above: https://github.com/n8n-io/n8n/pull/7643#discussion_r1393990223